services/transaction/tests/functional/multisig.create.test.js

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
```

Comment:
I'd recommend we remove the Shebang declaration as this file is not intended to be run as an independent shell script.
```suggestion
```